ui/src/config.js

```diff
@@ -36,6 +36,9 @@ export const appConfig = {
   privateDockerRegistries: process.env.REACT_APP_PRIVATE_DOCKER_REGISTRIES
     ? process.env.REACT_APP_PRIVATE_DOCKER_REGISTRIES.split(",")
     : [],
+  defaultDockerRegistry: process.env.REACT_APP_DEFAULT_DOCKER_REGISTRY
+    ? process.env.REACT_APP_DEFAULT_DOCKER_REGISTRY
+    : "docker.io", // User Docker Hub as the default
```

Comment:
`process.env.REACT_APP_DEFAULT_DOCKER_REGISTRY || "docker.io"` could be a better way for specifying default values